src/prog_models/prognostics_model.py

```diff
@@ -1142,7 +1137,7 @@ def calc_error(self, times: List[float], inputs: List[InputContainer], outputs:
             return calc_error.MAPE(self, times, inputs, outputs, **kwargs)
 
         # If we get here, method is not supported
-        raise ProgModelInputException(f"Error method '{method}' not supported")
+        raise ValueError(f"Error method '{method}' not supported")
```

Comment:
Should this be NotImplementedError? I've never used this particular error before, but based on your use of it above, maybe it would be good here